### DISCUSSION TITLE
how do you disable esp and fullbright?

### DISCUSSION THREAD

**Discussion by zx3800** [winstxnhdw/lc-hax#202] [Q&A] [answered]
i want to use this as a trolling device/utility when playing the game, but the game looks very very ugly to me with the esp and fullbright.
is there a way to disable it?

**Comment by GigglingMinister** [comment]
pretty sure you can disable them in loader.cs or something

**Comment by o777l** [comment, marked as answer]
Go into lc-hax/Scripts, open Loader.cs with text editor

Ctrl-F "LoadHaxModules” or just scroll to bottom

Put // in front of the module to disable

```
// Loader.AddHaxModules<ESPMod>();
// Loader.AddHaxModules<ClearVisionMod>();
```

Save
